frontend/src/app/containers/UpgradeWarningPage/stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import UpgradeWarningPage from './index';
+import LayoutWrapper from '../../components/LayoutWrapper';
+
+const baseProps = {
+  sendToGA: action('sendToGA'),
+  host: 'testpilot.firefox.com',
+  protocol: 'https:',
+  isMinFirefox: true
+};
+
+storiesOf('UpgradeWarning', module)
+  .addDecorator(withKnobs)
+  .addDecorator(story =>
+     <div className="blue" style={{ padding: '10px' }} onClick={action('click')}>
+       <div className="stars" />
+       <LayoutWrapper flexModifier="card-list">
+         {story()}
+       </LayoutWrapper>
+     </div>
+   )
+  .add('default', () => <UpgradeWarningPage {...baseProps} />)
+  .add('below minimum firefox', () => <UpgradeWarningPage {...baseProps} isMinFirefox={false} />)
+  .add('wrong protocol (http)', () => <UpgradeWarningPage {...baseProps} protocol={'http:'} />)
+  .add('developing test pilot', () => <UpgradeWarningPage {...baseProps} host={'example.com:8000'} />)
+  .add('unapproved hostname', () => <UpgradeWarningPage {...baseProps} host={'cyber.hell.scape'} />);
```

Comment:
😆

Reply:
Would still kind of like this to be just a simple flag prop like `isUnapprovedHost`, but wouldn't block merging this on that

Reply:
i agree, I'll finish this up rn